docs/how-to/specify-a-base.rst

```diff
@@ -0,0 +1,58 @@
+.. _how-to-specify-a-base:
+
+Specify a base
+==============
+
+All rocks are layered on top of a software base, which determines compatibility with
```

Comment:
what about `bare` rocks? They are built using a certain software base, but I would not say they are 'layered' on top of that. Layering has a very specific meaning for container images